proposals/lip-0037.md

```diff
@@ -1,6 +1,6 @@
 ```
 LIP: 0037
-Title: Use message tags and network identifiers for signatures
+Title: Use message tags and chain identifiers for signatures
 Author: Andreas Kendziorra <andreas.kendziorra@lightcurve.io>
 Discussions-To: https://research.lisk.com/t/use-message-tags-and-network-identifiers-for-signatures/
 Status: Draft
```

Comment:
Should we add that this LIP replaces LIP 9?

Reply:
Is there a standard way to do this in the header?

Reply:
See https://github.com/LiskHQ/lips/blob/main/proposals/lip-0001.md#lip-header-preamble

Reply:
added